using-mobile-extensions/mobile-core/README.md

```diff
@@ -26,15 +22,18 @@ Fore more information, see [Identity](identity/).
 
 ## Lifecycle
 
-For more information, see [Lifecycle](lifecycle/).
-
-## Signals
-
-For more information, see [Signals](signals/).
+Sessions contain information about the app's current lifecycle, such as device information, application install or upgrade information, session start and pause times, number of application launches, and additional context data that is provided by the developer through the `LifecycleStart` API. Session data is persisted, so it is available across application launches.
 
+For more information, see [Lifecycle](lifecycle/).
 
+## Rules Engine
 
+This Mobile Core contains a rule engine that interprets all tracking, PII, Signals, _Request Content_, and extension-specific events for the SDK. Users can create rules, which when matched, trigger a _Response Content_ event for the listening extension to handle the associated consequence\(s\).
```

Comment:
Is "This" necessary in this sentence? Can we just say "Mobile Core contains..."?